drivers/ethernet/Kconfig.native_posix

```diff
@@ -12,6 +12,27 @@ menuconfig ETH_NATIVE_POSIX
 	  a process in your host system.
 
 if ETH_NATIVE_POSIX
+config ETH_NATIVE_POSIX_STARTUP_AUTOMATIC
+	bool "Start network interface automatically"
+	help
+	  If set, then the native_posix ethernet driver tries to setup network
+	  interface by itself. This can be annoying as then you would need to
```

Comment:
I guess "annoying" is not the word we want to have in technical reference docs (where all this stuff goes, besides just "make menuconfig").

E.g., could say "This can be no what user wants and would require executing the zephyr.exe as a root user". Otherwise, calling to @dbkinder for suggestions.

Reply:
I agree with "annoying" wording issue. I see it more as a security risk/vulnerability.